### PR TITLE
[16.0][mig] purchase_blanket_order: Migration to 16.0

### PR DESCRIPTION
**Pull request by dsolanki-initos**
Depends on `Web/web_action_conditionable`

### CODE REVIEW

**Comment by SergiCForgeFlow** [comment]
@dsolanki-initos Are you still working on this PR? If so, could you add this commit please [#2048](https://github.com/OCA/purchase-workflow/pull/2048)

File: purchase_blanket_order/views/purchase_blanket_order_views.xml [hunk 1]
```diff
@@ -0,0 +1,439 @@
...
+                                force_save="1"
+                                attrs="{'readonly': [('state', 'in', ('done','expired'))]}"
+                            >
+                                <tree editable="bottom">
+                                    <field name="sequence" widget="handle" />
+                                    <field name="name" invisible="1" />
+                                    <field
+                                        name="product_id"
+                                        context="{'partner_id':parent.partner_id, 'quantity':original_uom_qty, 'company_id': parent.company_id}"
+                                        groups="base.group_multi_company"
```
Comment (JordiBForgeFlow) [inline purchase_blanket_order/views/purchase_blanket_order_views.xml:144]: Does not make sense to add this group.

File: purchase_blanket_order/views/purchase_blanket_order_views.xml [hunk 2]
```diff
@@ -0,0 +1,439 @@
...
+                                    />
+                                    <field
+                                        name="original_uom_qty"
+                                        string="Original Qty"
+                                        context="{'partner_id':parent.partner_id, 'quantity':original_uom_qty, 'company_id': parent.company_id}"
+                                        groups="base.group_multi_company"
```
Comment (JordiBForgeFlow) [inline purchase_blanket_order/views/purchase_blanket_order_views.xml:150]: Does not make sense to add this group.

File: purchase_blanket_order/views/purchase_blanket_order_views.xml [hunk 3]
```diff
@@ -0,0 +1,439 @@
...
+        <field name="model">purchase.blanket.order.line</field>
+        <field name="arch" type="xml">
+            <tree create="false">
+                <field name="sequence" widget="handle" />
+                <field name="name" invisible="1" />
+                <field name="order_id" />
+                <field
+                    name="product_id"
+                    context="{'partner_id':parent.partner_id, 'quantity':original_uom_qty, 'company_id': parent.company_id}"
+                    groups="base.group_multi_company"
```
Comment (JordiBForgeFlow) [inline purchase_blanket_order/views/purchase_blanket_order_views.xml:283]: Does not make sense to add

File: purchase_blanket_order/views/purchase_blanket_order_views.xml [hunk 4]
```diff
@@ -0,0 +1,439 @@
...
+                />
+                <field name="product_uom" invisible="1" />
+                <field name="price_unit" />
+                <field name="date_schedule" />
+                <field
+                    name="original_uom_qty"
+                    string="Original Qty"
+                    context="{'partner_id':parent.partner_id, 'quantity':original_uom_qty, 'company_id': parent.company_id}"
+                    groups="base.group_multi_company"
```
Comment (JordiBForgeFlow) [inline purchase_blanket_order/views/purchase_blanket_order_views.xml:292]: Does not make sense to add

**Review by JordiBForgeFlow — CHANGES_REQUESTED** [review]
Changes required.

**Pull request closed without merging**